src/js/jsx/sections/pages/LayerFace.jsx

```diff
@@ -193,15 +217,54 @@ define(function (require, exports, module) {
                 isDropTargetBelow = true;
             }
 
+            var depth = layerStructure.depth(layer);
+            var isLastInGroup = false;
+
+            if (layerStructure.byIndex(layerIndex - depth)) {
+                isLastInGroup = isChildOfSelected &&
+                    layerStructure.byIndex(layerIndex - 1).kind === layer.layerKinds.GROUPEND;
+            }
+
+            // Check to see if this layer is right before the end of nested group
+            var endOfGroupStructure;
+            
+            if ((layerIndex - 1) - (layerIndex - depth) > 0) {
```

Comment:
This is a weird condition. I'm not exactly sure what this is testing, but it looks like... 
```
(layerIndex - 1) - (layerIndex - depth) > 0
iff
(layerIndex - 1) - layerIndex + depth > 0
iff 
depth - 1 > 0
iff
depth > 1
```